.github/workflows/clean-rev-image.yml

```
@@ -0,0 +1,96 @@
+
+    # Set up Docker for Docker Hub login
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    # Cleanup images for user-service
+    - name: Cleanup user-service Docker tags
+      id: cleanup_user_service
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:3:1: THRESHOLD appears unused. Verify use (or export if used externally)

(shellcheck)